src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private Fruit fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, Fruit fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public FruitTransaction() {
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public Fruit getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(Fruit fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;

```

Comment:
In the setter method, add validation logic for the 'quantity' field to check if it is not negative.

Reply:
ok, done